content/en/releases/download.md

```diff
@@ -3,6 +3,8 @@ title: Download Kubernetes
 type: docs
 ---
 
+# Core Kubernetes components
```

Comment:
We shouldn't have merged this - level 1 headings aren't allowed in our Markdown.

Reply:
Follow-up in https://github.com/kubernetes/website/pull/32895